jpo-ode-svcs/src/test/java/us/dot/its/jpo/ode/coder/stream/LogFileToAsn1CodecPublisherTest.java

```diff
@@ -227,7 +228,8 @@ public void testPublishDriverAlertLogFile() throws Exception {
             (byte) 0xa4, (byte) 0xa1, (byte) 0x5c, (byte) 0xce, // 1.1 longitude
             (byte) 0x67, (byte) 0x06, (byte) 0x00, (byte) 0x00, // 1.2 elevation
             (byte) 0x04, (byte) 0x00, // 1.3 speed
-            (byte) 0x09, (byte) 0x27, // 1.4 heading
+            (byte) 0x09, (byte) 0x27, // 1.4 heading         assertEquals(expected, data.toJson());
```

Comment:
Is this commented assert needed?

Reply:
Nope, must just be a remnant from some testing.